src/pages/elements/Service.js

```diff
@@ -40,16 +50,86 @@ const EscalationPolicy = ({app}) => {
   </React.Fragment>
 }
 
-function Service({ service, reports, documents }) {
-  const headerFormat = value => <Table.Heading>{value}</Table.Heading>;
-  const cellFormat = value => <Table.Cell>{value}</Table.Cell>;
-  const linkFormat = url => value => (
-    <a href={`${url || ''}${value}`} target="_blank" rel="noopener noreferrer">
-      {value}
-    </a>
-  );
-  const emptyFormat = value => value || '-';
-  const booleanFormat = (t, f) => value => (value ? t : f);
+// displays the list of Saas Deploy Jobs
+const SaasDeployJobs = ({f, settings}) => {
+  const job_template_name = settings[0].saasDeployJobTemplate;
+  var job_lst = [];
+  var job_name_lst = [];
+  var job_name;
+  for (var template of f.resourceTemplates) {
+    for (var target of template.targets) {
+      job_name = job_template_name + '-' + f.name + '-' + target.namespace.environment.name;
+      if (!job_name_lst.includes(job_name)) {
+        job_lst.push(<li><a href={f.instance.serverUrl + "/job/" + job_name + "/api/json?tree=builds[timestamp,result]"}> {job_name} </a></li>);
```

Comment:
We can use the direct job url here, no need for using api url

Reply:
Gotcha! It makes sense. Updated!